src/routes/transactions/transactions-view.controller.spec.ts

```diff
@@ -194,7 +195,9 @@ describe('TransactionsViewController tests', () => {
           },
           receiver: order.receiver,
           owner: order.owner,
-          fullAppData: JSON.parse(order.fullAppData),
+          fullAppData:
+            typeof order.fullAppData === 'string' &&
+            JSON.parse(order.fullAppData),
```

Comment:
As before, this can now return `false` and we should consider using a ternary.

Reply:
I've added a cast here (see https://github.com/safe-global/safe-client-gateway/pull/1533#discussion_r1624713408) 

Changed in [dbc6503](https://github.com/safe-global/safe-client-gateway/pull/1533/commits/dbc6503ff69aca7d3acde6c5b05c48da6927a29d)